index.js

```diff
@@ -1,13 +1,99 @@
+const $pModal = document.querySelector('#p-modal');
+const $imgModal = document.querySelector('#img-modal');
+const $liModal = document.querySelectorAll('.li-modal');
+const $openModal = document.querySelectorAll('.button-project');
+const $closeModal = document.querySelector('.close-button');
+const frstImagePath = '/resources/work-section/masks/Snapshoot PortfolioTonic-mask.png';
+const scndImagePath = '/resources/work-section/masks/Snapshoot PortfolioMulti-Post-mask.png';
+const thrdImagePath = '/resources/work-section/masks/Snapshoot PortfolioTonic2-mask.png';
+const frthImagePath = '/resources/work-section/masks/Snapshoot PortfolioMulti-Post2-mask.png';
+let scrollEnabled = true;
+const projectsInfo = [{
+  name: 'Tonic',
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  technologies: ['HTML', 'CSS', 'Javascript'],
+  image: frstImagePath,
+}, {
+  name: 'Multi Post Stories',
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  technologies: ['HTML', 'CSS', 'Javascript'],
+  image: scndImagePath,
+}, {
+  name: 'Tonic',
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  technologies: ['HTML', 'CSS', 'Javascript'],
+  image: thrdImagePath,
+}, {
+  name: 'Multi-Post Stories',
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  technologies: ['HTML', 'CSS', 'Javascript'],
+  image: frthImagePath,
+}];
 
-let div = document.createElement('div');
-div.id = 'content';
-div.className = 'note';
+function selectProject(buttonIndex) {
+  const project = projectsInfo[buttonIndex];
+  $pModal.textContent = project.description;
+  $imgModal.setAttribute('src', project.image);
+  $liModal.forEach((element, index) => {
+    element.textContent = project.technologies[index];
```

Comment:
- [ ] Kindly adhere to the following suggestions to match the design 💯.

- Your design
<img width="778" alt="image" src="https://github.com/SaveryIV/SaveryIV.github.io/assets/41163571/31d81281-7a7c-435c-bb4a-839a62d2f039">


- Figma design
<img width="389" alt="image" src="https://github.com/SaveryIV/SaveryIV.github.io/assets/41163571/faaa97f0-f6e1-44cc-a2f1-11cae557e317">

- The `See live` and `See source` icons respectively.
- The alignment between the languages list and the buttons.
- The spacing between the lorem ipsum texts and what is by its right.
- The little grey line beneath the languages list.
- The space between the language items is too much compared to the design.